typings/custom-oppia-defs.d.ts

```diff
@@ -0,0 +1,22 @@
+// These declarations are required to avoid compile errors while compiling
```

Comment:
We will not need this if we compile the entire typescript each time instead of compiling single files that have changed but that will slow down the build process.